### PR TITLE
remove special cases for featureset in rendering

### PR DESCRIPTION
**Pull request by deads2k**
This completes the reset for rendering featureset sensitive manifests based on the FeatureGate manifest provided by the cluster-config-operator.

### CODE REVIEW

**Comment by patrickdillon** [comment]
> > * could not run steps: step installer-artifacts failed: error occurred handling build installer-artifacts: build didn't start running within 30m0s (phase: Running):
> >   Found 17 events for Pod installer-artifacts-build:
> 
> /retest

Revert for the cause is being floated here: https://github.com/openshift/ci-tools/pull/3409

I'm also floating https://github.com/openshift/installer/pull/7159 for discussion.

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -89,10 +92,6 @@ then
 	if [ -f "$PWD/manifests/cloud-provider-config.yaml" ]; then
 		ADDITIONAL_FLAGS+=("--cloud-provider-config-input-file=/assets/manifests/cloud-provider-config.yaml")
 	fi
-	{{- if .FeatureSet }}
-	ADDITIONAL_FLAGS+=("--feature-set={{.FeatureSet}}")
-	{{- end}}
```
Comment (patrickdillon) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:94]: Deleting these three lines is not covered by #7152

**Review by patrickdillon — COMMENTED** [review]
I'm a bit confused about the relationship between this and #7152. Do we still need this?

**Comment by JoelSpeed** [comment]
@deads2k This broke the feature gate manifest generation process and I think has broken installations.

While the manifest file path you removed here has a full path when it [writes out](https://github.com/openshift/cluster-config-operator/blob/b4a0649a59cc58bbba8709595ab1ed50625794a0/pkg/cmd/render/render.go#L148), the paths for the feature gate files found by the generic/path walk approach we have, return a relative path to the rendered path, so when we write out using [OriginalFileName](https://github.com/openshift/cluster-config-operator/blob/b4a0649a59cc58bbba8709595ab1ed50625794a0/pkg/cmd/render/render.go#L183) it writes the file directly to CWD rather than the original path. Any idea why original file name shouldn't be the absolute path?

**Pull request merged**